go.mod

```diff
@@ -483,7 +483,7 @@ require (
 require (
 	github.com/coder/agentapi-sdk-go v0.0.0-20250505131810-560d1d88d225
 	github.com/coder/aisdk-go v0.0.9
-	github.com/coder/preview v1.0.3-0.20250701142654-c3d6e86b9393
+	github.com/coder/preview v1.0.3-0.20250713201143-17616ecf763a
```

Comment:
NB: We must not merge this PR until we've merged https://github.com/coder/preview/pull/149 and updated this dependency in kind.

Reply:
done.